llvm/test/CodeGen/AArch64/cmpbr-reg-imm.ll

```diff
@@ -0,0 +1,402 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple arm64-apple-ios -mattr +cmpbr -verify-machineinstrs -o - < %s | FileCheck %s
+
+define void @cbgt.i32(i32 %a)  {
```

Comment:
It is best to remove the '.'s from function names.

Reply:
Done